vcpkg/contributing/removed_vcpkg_catalog_release_process.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Just curious why you are introducing these files?
Your goal was to delete them, not rename them, yes?
Since they were already deleted, that's water under the bridge. I'd remove these renamed files and just focus on the redirect file changes.